.github/workflows/automated-zip-to-pr.yml

```diff
@@ -0,0 +1,124 @@
+name: Automated Zip to PR
+
+on:
+  issues:
+    types: [ "opened", "edited" ]
+
+  issue_comment:
+    type: [ "created", "edited" ]
+
+jobs:
+
+ extrakt:
+
+    # We don't do anything on pull requests, which are also considered issues
+    if: ${{ !github.event.issue.pull_request }}
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+      # Only do these if the target is the ISSUE not a COMMENT on the issue
+      - name: Extract Zip URL (body)
+        uses: actions-ecosystem/action-regex-match@v2
+        if: ${{ !github.event.comment.body }}
+        id: match-url
+        with:
+          text: ${{ github.event.issue.body }}
+          regex: '\((.*?\.zip)\)'
```

Comment:
The action triggers for any new issue created and creates PRs if it has a Zip attached.

Trigger the action on issues that are having a special label. like `New Theme` may be.